dotnet/src/Connectors/Connectors.AzureAIInference/Extensions/AzureAIInferenceServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Azure.AI.Inference;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.AzureAIInference;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Sponsor extensions class for <see cref="IKernelBuilder"/>.
+/// </summary>
+public static class AzureAIInferenceServiceCollectionExtensions
+{
+    /// <summary>
+    /// Adds the Azure AI Inference chat completion service to the list.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> instance to augment.</param>
+    /// <param name="modelId">Target Model Id for endpoints supporting more than one model</param>
+    /// <param name="apiKey">API Key</param>
+    /// <param name="endpoint">Endpoint / Target URI</param>
+    /// <param name="serviceId">A local identifier for the given AI service</param>
+    /// <returns>The same instance as <paramref name="services"/>.</returns>
+    public static IServiceCollection AddAzureAIInferenceChatCompletion(
+        this IServiceCollection services,
+        string? modelId = null,
+        string? apiKey = null,
+        Uri? endpoint = null,
+        string? serviceId = null)
```

Comment:
In similar `IKernelBuilder` extension method we also accept optional `HttpClient`, should we add it here as well? 

Also, in vector store connectors there is a following pattern - we define extension method for `IServiceCollection`, and then we use this extension method for `IKernelBuilder` by using one line `builder.Services.AddAzureAIInferenceChatCompletion`. It works well, because you don't need to define the same logic in both places, you can use `IServiceCollection` extension method for `IKernelBuilder` extension method. I'm wondering if we can use the same approach here?